### ISSUE TITLE
Provisioning of isolated node failed.

### ISSUE THREAD

**Issue by tchyou** [ansible/awx#2588]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
  - UI
 
##### SUMMARY
after modifying inventory (/installer/inventory) to add isolated node group and reinstalling, no isolated node group can be shown.

##### ENVIRONMENT
* AWX version: 2.1.0
* AWX install method: docker on linux
* Ansible version:  2.7.1
* Operating System: Centos 7.5
* Web Browser: Google Chrome

##### STEPS TO REPRODUCE

1. Install node B as AWX isolation node on docker B.

2. modifying node A's inventory (/installer/inventory) to add isolated node group including node B.

3. Install node A as AWX Controller on docker A.

4. Create and import SSH credential and ensure docker A can SSH docker B without password.

5. Open node A's AWX admin UI. 

##### EXPECTED RESULTS

Isolated Group with node B should be shown.

##### ACTUAL RESULTS

No Isolated Group is shown.

##### ADDITIONAL INFORMATION

NA

**Comment by weichih-c** [comment]
I met the same issue too!

I fetched the latest image from docker hub (including Task:2.1.0, Web:2.1.0, rabbitmq:3.7.4, pgsql:9.6, memcached:latest) and built the docker-isolated image(in tool/docker-isolated).

I've run all docker image from docker-hub on one machineA through install.yml in installer dir (just like install.md done), and run the docker-isoloated image through docker-compose.

Besides, I installed awx images on another machine B and set the isolated-group info in inventory file (in installer directory). Just following the [official site](https://docs.ansible.com/ansible-tower/latest/html/administration/clustering.html)
 
But in machine B's web UI, i could not see any other instance group but only tower in it.
